content/pt/docs/reference/glossary/node.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Node
+title: Nó
 id: node
 date: 2020-04-19
 full_link: /docs/concepts/architecture/nodes/
 short_description: >
-  Um Node é uma máquina de trabalho no Kubernetes.
+  Um Nó é uma máquina de trabalho no Kubernetes.
 
 aka: 
 tags:
 - fundamental
 ---
- Um Node é uma máquina de trabalho no Kubernetes.
+ Um Nó é uma máquina de trabalho no Kubernetes.
 
 <!--more--> 
 
-Um Node pode ser uma máquina virtual ou física, dependendo do cluster. Possui daemons ou serviços locais necessários para executar {{< glossary_tooltip text="Pods" term_id="pod" >}} e é gerenciado pelo {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}}. Os daemons em um Node incluem  {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}, {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} e um contêiner runtime implementando o {{< glossary_tooltip text="CRI" term_id="cri" >}} como por exemplo o {{< glossary_tooltip term_id="docker" >}}.
\ No newline at end of file
+Um Nó pode ser uma máquina virtual ou física, dependendo do cluster. Possui daemons ou serviços locais necessários para executar {{< glossary_tooltip text="Pods" term_id="pod" >}} e é gerenciado pelo {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}}. Os daemons em um Node incluem  {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}, {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} e um contêiner runtime implementando o {{< glossary_tooltip text="CRI" term_id="cri" >}} como por exemplo o {{< glossary_tooltip term_id="docker" >}}.
```

Comment:
Hi Ricardo, to keep the context, please, change this "plano de controle" to "ambiente de gerenciamento". Great job and very thanks!